lib/sidekiq/throttled/strategy/concurrency.rb

```diff
@@ -46,20 +54,18 @@ def throttled?(jid, *job_args)
           return false unless job_limit
           return true if job_limit <= 0
 
-          keys = [key(job_args)]
-          argv = [jid.to_s, job_limit, @ttl, Time.now.to_f]
+          keys = [key(job_args), backlog_info_key(job_args)]
+          argv = [jid.to_s, job_limit, @lost_job_threshold, Time.now.to_f]
 
-          Sidekiq.redis { |redis| 1 == SCRIPT.call(redis, keys: keys, argv: argv) }
+          Sidekiq.redis { |redis| (1 == SCRIPT.call(redis, keys: keys, argv: argv)) }
```

Comment:
Parenthesis are redundant here
```suggestion
          Sidekiq.redis { |redis| 1 == SCRIPT.call(redis, keys: keys, argv: argv) }
```